Update switch-statements-practice-answers.md

ZThatWhiteGuyZ opened the pull request:
Reformatted the answer that had an issue. Put the answer in a code block.

Commit:
Update switch-statements-practice-answers.md

Reformatted the answer that had an issue. Put the answer in a code block.

jerrettl commented:
The code for the answer doesn't have newlines, you might want to edit that so it matches with the output

Commit:
Answer wording correction

Changed the wording in the last problem to be dispatcher instead of operator. If there's a time police emergency, dispatcher is what you want. Operator would be for more internal things.

Commit:
Update switch-statements.md

I was looking at the file and found white space. Jerrett will appreciate this commit.

Commit:
Update switch-statements.md

Added a new section and several subsections (and a sub-sub section). Very much a work in progress, but open to feedback on my newest section.

jerrettl commented:
The changes in this issue may want to be split into a separate PR, as outside of the changes made to the formatting, there seems to be additional in-progress developments being made to the switch statements page.

ZThatWhiteGuyZ committed:
Merge branch 'main' into switch-statements-issue-fix

ZThatWhiteGuyZ committed:
Update switch-statements-practice-answers.md

This should resolve the very cryptic issue that I posted. The answer has been cleaned of its whitespace, and I added a note about the outputs. Seeing as the inputs given are separated by whitespace to emphasize they are different runs, I noted that each output should appear on a new line. Please look through and check my answer key. If it's good, it should be merge ready.

ZThatWhiteGuyZ committed:
Switch-statements.md update: Duff's device and loop unrolling

This is a flesh-out of the "Weird switch statement" section. Please read through it for readability and understanding. This is a first draft, so please, let me have it.

ZThatWhiteGuyZ committed:
Update to switch-statements.md: Arithmetic and Conditionals in Switch Statements

I worked out another section. I intentionally broke these into two requests. The final section left is Switch Statement Errors. Please give this a read through. First draft type beat.
